Dockerfile

```diff
@@ -4,10 +4,11 @@ MAINTAINER ilari.makela@wunderkraut.com
 # Update the package repository and install applications
 RUN apk --no-cache --update add mariadb && \
     rm -rf /tmp/* && \
-    rm -rf /var/cache/apk/* && \
-    mysql_install_db
+    rm -rf /var/cache/apk/*
+
+RUN mysql_install_db && \
+    chown -R mysql /var/lib/mysql
 
-VOLUME /var/lib/mysql
```

Comment:
I am finding that the "/var/lib/mysql" volume loses all of the permission changes needed for startup.  I am not sure why.